stock_picking_return_refund_option/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2018 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    'name': 'Stock Picking Return Refund Option',
+    'summary': 'Update the refuund options in pickings',
+    'version': '11.0.1.0.0',
+    'development_status': 'Production/Stable',
+    'category': 'Sales',
+    'website': 'https://github.com/OCA/account-invoicing',
+    'author': 'Tecnativa, Odoo Community Association (OCA)',
+    'license': 'AGPL-3',
+    'application': False,
+    'installable': True,
+    'depends': [
+        'sale_stock',
```

Comment:
It should depend on `stock_account`, as the field is now defined in that module and for allowing to use this option also on purchases.

Reply:
The method _get_delivered_qty declared on sale_stock is used..